install_scripts/IBEX_upgrade.py

```diff
@@ -21,13 +22,13 @@
     parser.add_argument("--quiet", default=False, action="store_true",
                         help="Do not ask any questions just to the default.")
 
-    parser.add_argument('upgrade_type', choices=['training', 'demo'], help="What upgrade should be performed")
+    parser.add_argument('upgrade_type', choices=['training', 'demo', 'instrument_update'], help="What upgrade should be performed")
```

Comment:
Should this be `instrument`, `instrument_upgrade`, or `instrument_update`? We should be clear and consistent on terminology. My guess is you did it like this because it's fundamentally different from the other types since it doesn't install new versions. However, @John-Holt-Tessella is of the thought that this is our eventual entry point for the proper upgrade, in which case it should presumably just be `instrument`?

Reply:
Yes you're right I did it because the scope of this upgrade is quite different to the others.

It won't be hard to change the name if the scope does change in the future.